src/lesson8/participant/Human.java

```diff
@@ -0,0 +1,54 @@
+package lesson8.participant;
+
+public class Human extends Participant{
```

Comment:
same here, parent method will be called, so you don't need to create the same method setMaxJump ...